src/gui/transferlistmodel.h

```diff
@@ -39,6 +40,47 @@ namespace BitTorrent
     class TorrentHandle;
 }
 
+// convenience class to read text colors from qss since qss doesn't work with qobject
+class TransferListTextColors : public QWidget
```

Comment:
Maybe you only need to inherit QObject instead of QWidget?

Reply:
@Chocobo1, see comment above.